frontend-cert/react-projects/reusable-footer/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should have a `div` with an `id` of `root`.
+1. You should have a `script` tag that specifies you are using `babel` to compile `JSX`.
+1. You should have a `Footer`component.
+1. Your `Footer` component should return a `footer` element.
+1. The `footer` should have:
+
+- Three unordered lists each with two list items.
+- A copyright paragraph.
+- Three icons for social media.
+
+1. You should select the `#root` div and create a root element to support rendering.
+1. You should render the `Footer` component to the `root` element.
```

Comment:
we don't need to test for this because it will already be apart of the opening boilerplate